src/backend/src/services/notifications.services.ts

```diff
@@ -77,7 +77,7 @@ export default class NotificationsService {
     const promises = Array.from(teamTaskMap).map(async ([slackId, tasks]) => {
       const messageBlock = tasks
         .map((task) => {
-          const daysUntilDeadline = daysBetween(task.deadline, new Date());
+          const daysUntilDeadline = daysBetween(task.deadline!, new Date());
```

Comment:
add a comment explaining why were able to force unwrap here because of the prisma call earlier